examples/stickysession/nginx/README.md

```diff
@@ -0,0 +1,58 @@
+# Sticky Session
+
+This example demonstrates how to Stickness in a Ingress.
```

Comment:
Hum, actually this session affinity is through cookie. The sticky module captures the cookie, and routes the request based on this. I didn't deployed more than one NGINX to check if the hash that the cookie assigns keeps the same, between them, but I think it did, as the hash is based on the upstream name.